packages/menu/src/useMenu.ts

```diff
@@ -448,19 +476,13 @@ export const useMenu = <T extends HTMLElement = HTMLElement, M extends HTMLEleme
           ...(nextSelection && { selectedItems: nextSelection })
         };
 
-        if (item.href) {
-          if (key === KEYS.SPACE) {
-            event.preventDefault();
```

Comment:
Anchors should be also triggered with `KEYS.ENTER`.

Reply:
Oh shoot, thought we fixed that. Thanks!